acpi/src/fadt.rs

```diff
@@ -128,3 +131,27 @@ impl Fadt {
         }
     }
 }
+
+/// Information about the ACPI Power Management Timer (ACPI PM Timer).
+pub struct PmTimer {
+    /// An I/O space address to the register block of ACPI PM Timer.
+    pub io_base: u32,
+    /// This field is true if the hardware supports 32-bit timer, and false if the hardware
+    /// supports 24-bit timer.
+    pub supports_32bit: bool,
+}
+impl PmTimer {
+    /// Creates a new instance of `PmTimer`.
+    pub fn new<H>(tables: &AcpiTables<H>) -> Result<PmTimer, AcpiError>
```

Comment:
The PM timer is an optional feature (if both `PM_TMR_BLK` and `X_PM_TMR_BLK` are zero), so this should probably return `Result<Option<PmTimer>, AcpiError>`.